README.md

```diff
@@ -1,9 +1,23 @@
+## Ruff for VS Code - Pre-Release Information
+
+This is a pre-release edition of Ruff for VS Code that uses our new, Rust-based LSP - `ruff server`.
+
+At the moment, the pre-release has the following known limitations:
+
+- Extension settings are not yet supported, and many will not be needed after the transition to the new server is finished. For example, `lint.args` / `format.args` will be replaced in the future with specific configuration fields for the linter and formatter.
+- Commands like `Fix all` and `Quick Fix` have not yet been implemented. (code actions should still work, though)
+- Hierarchial configuration for individual files is not yet supported. At the moment, the language server uses the `ruff.toml` / `pyproject.toml` at the workspace root to configure the formatter and linter.
+- Jupyter Notebook files are not supported yet.
+- Using local Ruff binaries is not yet supported. At the moment, the extension will always use the bundled Ruff binary. (`v0.3.3`)
+
+To use the new Rust-based language server, you'll need to enable the "Experimental Server" setting and reload the extension. Please report any issues you have to the Github issue tracker.
```

Comment:
So, to confirm, this is only ever intended to render in the pre-release version?

Reply:
Correct - I think a feature flag makes sense for pre-release versions, but once we've deemed it 'stable enough' it should become the default in newly-released regular versions (ideally we'll have settings migrations / deprecations by then).